js/layout-javascript/news-feed-code.js

```diff
@@ -203,16 +207,32 @@ DynamicList.prototype.attachObservers = function() {
         sortField: _this.sortField
       });
     })
-    .on('click', '.apply-filters', function() {
+    .on('click keydown', '.apply-filters', function(event) {
+      if (!_this.Utils.accessibilityHelpers.isExecute(event)) {
+        return;
+      }
+
+      $('.fa-sliders').focus();
```

Comment:
Fix all similar errors in the PR. There should never be any `$(selector)` directly querying an element within the container.

```suggestion
      _this.$container.find('.fa-sliders').focus();
```

Reply:
@tonytlwu fixed